painters/train_cnn.py

```diff
@@ -26,6 +34,13 @@
 SOFTMAX_LAYER = 55
 SOFTMAX_SIZE = 1584
 
+# Specify hyperparameters for this trial.
+kernel_size = pedl.get_hyperparameter("kernel_size")
+dropout = pedl.get_hyperparameter("dropout")
+pool_size = pedl.get_hyperparameter("pool_size")
+L2_REG = pedl.get_hyperparameter("L2_REG")
+lr = pedl.get_hyperparameter("lr")
```

Comment:
I would generally warn against creating global variables to store hyperparameters, but since this file is already doing it for other hyperparameters, i'll allow it :P

Please use capitalization for the variable names to indicate they are globals.